### PR TITLE
Made Changes to Exif2TimeStream

### PR DESCRIPTION
**Pull request by jbraiuka**
Uses both exif read and pexif. 
 - Pexif for writing to jpeg exif file formats
 - exif read for reading from raw images
Resize uses Scikit image

Wrote tests (Cant work out how to add them to be run)

### CODE REVIEW

**Comment by gdunstone** [comment]
You should use vips for image resize.

File: exif2timestream.py [hunk 1]
```diff
@@ -236,24 +246,101 @@ def __call__(self, x):
         return None
 
 
+def resize_img(filename, to_width):
```
Comment (kdmurray91) [inline exif2timestream.py]: @jbraiuka you should write a test for this function

File: exif2timestream.py [hunk 2]
```diff
@@ -236,24 +246,101 @@ def __call__(self, x):
...
+    # read in old exxif data
+    exif_source = pexif.JpegFile.fromFile(filename)
+    # Save image
+    io.imsave("test" + filename, img)
+    # Write new exif data from old image
+    exif_dest = pexif.JpegFile.fromFile("test" + filename)
+    exif_dest.exif.primary.ExtendedEXIF.DateTimeOriginal = exif_source.exif.primary.ExtendedEXIF.DateTimeOriginal
+    exif_dest.writeFile("test" + filename)
+
+def get_time_from_filename(filename, mask):
```
Comment (kdmurray91) [inline exif2timestream.py]: And this one

File: exif2timestream.py [hunk 3]
```diff
@@ -236,24 +246,101 @@ def __call__(self, x):
...
+        # If we cant convert it to the date, then go to the next item on the list
+        except ValueError:
+            continue
+    # If we cant match anything, then return null
+    return None
+
+
+
+def write_exif_date(filename, date_time):
+    try:
```
Comment (kdmurray91) [inline exif2timestream.py]: And this one

**Comment by jbraiuka** [comment]
I Fucked up the branches. See the other pull request. #22

**Pull request closed without merging**